src/components/services-view/services-view.controller.ts

```diff
@@ -65,12 +65,17 @@ export class ServicesViewController implements angular.IController {
     }
 
     this.ctrl.currentFilter = category;
-    this.ctrl.currentSubFilter = (this.ctrl.subCategories.length === 1) ? this.ctrl.subCategories[0].id : (subCategory || 'all');
+    this.ctrl.currentSubFilter = (this.ctrl.subCategories.length === 1) ? this.ctrl.subCategories[0].id : (subCategory || null);
```

Comment:
Hi, these [changes](https://github.com/dtaylor113/origin-web-catalog/commit/dee94e4a10047a83aea2ec3d72e424ef98bc9cb6) seem to be closer to what we want.  Please note that a few unit tests will need to be updated due to the initial hiding of items -unit tests are expecting them to be visible when clicking on a main category.